FirebaseRemoteConfig/Tests/Unit/RCNConfigContentTest.m

```diff
@@ -262,4 +291,38 @@ - (void)testCopyFromDictionaryUpdatesActiveConfig {
                         [_configContent.activeConfig[@"dummy_namespace"][@"new_key"] dataValue]);
 }
 
+- (void)testCheckAndWaitForInitialDatabaseLoad {
```

Comment:
@ArtVolkov  Great, thank you for adding the test! Does it fail if running without the code change?

Reply:
@maksymmalyhin. Yes, two falls were here with old code: one for each caveat in PR description.

Reply:
Great, thank you for confirming.